### PR TITLE
Facehuggers can no longer latch onto mobs without a head

### LINKED ISSUE #21437: Facehuggers can latch onto bodies without a head

### PR DESCRIPTION
**Pull request by Cyberboss**
Fixes #21437

:cl: Cyberboss
fix: Facehuggers can no longer latch onto mobs without a head... How the fuck did you get a living mob without a head?
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/alien/special/facehugger.dm [hunk 1]
```diff
@@ -111,6 +111,9 @@ var/const/MAX_ACTIVE_TIME = 400
 		return 0
 	if((!iscorgi(M) && !iscarbon(M)) || isalien(M))
 		return 0
+	var/mob/living/carbon/Carb = M
+	if(!Carb.head)
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/alien/special/facehugger.dm]: this will runtime on corgis

Comment (Cyberboss) [inline code/modules/mob/living/carbon/alien/special/facehugger.dm]: REEEEE

**Comment by XDTM** [comment]
The most common way of getting headless living carbons is changelings, since they don't need heads to live but people decapitate them nonetheless.

**Comment by WJohn** [comment]
i wonder why changelings don't regrow their heads after decapitation. it's kind of a dead give away if someone's running around headless after regeneration. i'm guessing it has to do with creating a new brain and some ckey bullshit that's too much trouble to do.

**Comment by PKPenguin321** [comment]
changelings can regenerate their head with their regenerate power, it's a thematic thing and it's actually really cool

**Pull request merged**